EIPS/eip-4834.md

```diff
@@ -34,24 +33,24 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Solidity Interface with NatSpec & OpenZeppelin v4 Interfaces (also available at [IDomain.sol](../assets/eip-4834/IDomain.sol)):
 
-```sol
+```solidity
 interface IDomain is IERC165 {
```

Comment:
Since you are using ERC165, consider write down the identifier?
```
the ERC-165 identifier for this interface is 0xNNNNNNNN.
```

Reply:
Sure thing. Finding out the identifier of an interface is a pain though 💀

Reply:
I've put a placeholder there (which is technically valid). If I have spare time, I'll go actually evaluate it.